nextcord/ext/commands/core.py

```diff
@@ -1202,13 +1202,11 @@ class GroupMixin(Generic[CogT]):
         Whether the commands should be case insensitive. Defaults to ``False``.
     """
 
-    def __init__(self, *args: Any, **kwargs: Any) -> None:
-        case_insensitive = kwargs.get("case_insensitive", False)
+    def __init__(self, *, case_insensitive: bool = False) -> None:
         self.all_commands: Dict[str, Command[CogT, Any, Any]] = (
             _CaseInsensitiveDict() if case_insensitive else {}
         )
         self.case_insensitive: bool = case_insensitive
-        super().__init__(*args, **kwargs)
```

Comment:
#695 broke `case_insensitive` since it is getting passed to Client as part of kwargs here.

Removing this call to `Client.__init__` fixes that bug, so no suggestion here, just saying priority might be a bit higher for this change.